src/charm.py

```diff
@@ -25,6 +26,12 @@ def __init__(self, *args: typing.Any) -> None:
         """
         super().__init__(*args)
         self.framework.observe(self.on.config_changed, self.config_service)
+        self.ingress = IngressPerAppRequirer(
+            self,
+            port=8000,
+            host=f"{self.app.name}-endpoints.{self.model.name}.svc.cluster.local",
```

Comment:
Can you add a comment for this that point to the bug that forces us to use this trick ?

Reply:
Added